components/IscnRegisterForm.vue

```diff
@@ -850,6 +866,17 @@ export default class IscnRegisterForm extends Vue {
     return 'CreativeWork'
   }
 
+  get defaultLanguage() {
+    const containsOnlyASCII = (str: string) => /^[ -~]*$/.test(str);
+
+    const isEpubInChinese = this.epubMetadata?.language === 'zh';
```

Comment:
Should we respect if epub metadata language is explicitly set to en?

Reply:
[🎨 Prioritize language from epubMetadata](https://github.com/likecoin/app-like-co/pull/471/commits/7f5a5a736db1c90c64104b9d9de553a2262d92d4)